requirements.txt

```diff
@@ -1,5 +1,8 @@
 six>=1.9.0
 docutils
 pyyaml
+ruamel.yaml
+requests
 packaging
 markupsafe
+lxml
```

Comment:
Can we use extras (https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies) to make ruamel.yaml and lxml optional? Maybe you get them with ``pip install galaxy-lib[mulled]``? Also can you conditionally import them and throw errors at runtime if they aren't there?

We don't have to make the dependencies optional in Conda, we can add them to the recipe there - I'd just like to stay as light-weight as possible for PyPI based installs. I treat other Galaxy dependencies the same way when they are in galaxy-lib (e.g. https://github.com/galaxyproject/galaxy-lib/blob/master/galaxy/objectstore/s3.py#L15).

Reply:
I took another look at the code and realized they can be removed - ruamel.yaml doesn't do anything pyYAML can't do and HTML can be parsed using the standard library instead of lxml (with a few extra lines of code). Thanks for the comments!